data/json/items/magazine/762.json

```diff
@@ -108,7 +108,7 @@
     "looks_like": "stanag30",
     "type": "MAGAZINE",
     "name": { "str": "vz. 58 10-round magazine" },
-    "description": "A surplus Czech military magazine for the vz. 58 assault rifle, which has been modified to only be capable of holding 10 rounds in place of its standard capacity of 30 cartridges.",
+    "description": "A surplus Czechoslovak military magazine for the vz. 58 assault rifle, which has been modified to only be capable of holding 10 rounds in place of its standard capacity of 30 cartridges.",
```

Comment:
Czechoslovakian is better i think

Reply:
Wikipedia lists the name of the military serving the state of Czechoslovakia as the "Czechoslovak military". From this we can infer that the correct word to use about something of or relating to Czechoslovakia is "Czechoslovak", not "Czechoslovakian".